CHANGELOG

```diff
@@ -1,3 +1,13 @@
+Buku v3.X
+Unreleased
+
+What's in?
+- Add --cached to completion scripts
```

Comment:
We can omit this one. The idea is all options should be present in the completion scripts.